docker-compose.yml

```diff
@@ -54,6 +54,12 @@ services:
       # Comment out ports: when using behind a reverse proxy , enable networks: etc.
       ports:
         - 5000:5000
+      deploy:
+        resources:
+          limits:
+            memory: 1G
```

Comment:
this is super cool, however, even better is to leave this commented out `#` so that people can install it

juuuust incase someone really needs it to run with that much ram, atleast they can find an easy solution